stdgo/reflect/Reflect.hx

```diff
@@ -25,63 +25,14 @@ enum GT_enum {
 	GT_string;
 	GT_unsafePointer;
 	GT_chan(elem:GT_enum);
-	GT_interface;
+	GT_interface(name:String,type:GT_enum);
 	GT_ptr(elem:GT_enum);
 	GT_slice(elem:GT_enum);
 	GT_array(elem:GT_enum, len:Int);
 	GT_func(input:Array<GT_enum>, output:Array<GT_enum>);
 	GT_map(key:GT_enum, value:GT_enum);
-	GT_struct(fields:Array<StructField>);
-	// go2hx Type system internal enum, no Kind equivalent
-	GT_namedType;
-}
-
-
-// TODO - make a go2hx standard class, so that it can be accurately reflected upon
-typedef Method = {
```

Comment:
This is the data structure returned by  `reflect.Type.method()`, it will be needed somewhere in the code.